rotkehlchen/tests/unit/decoders/test_stakedao.py

```diff
@@ -157,3 +157,120 @@ def test_claim_multiple(ethereum_inquirer, ethereum_accounts):
+            counterparty=CPT_STAKEDAO,
+            address=vault,
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x5bAaC7ccda079839C9524b90dF81720834FC039f']])
+def test_withdraw(ethereum_inquirer, ethereum_accounts):
+    tx_hex = deserialize_evm_tx_hash('0x1f0b98aa12fb35df17801ddfbbc0c2979ec611b50311535bad92ab5ec54f65f9')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = ethereum_accounts[0]
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hex)
+    gas_amount, cvr_reward, cvx_reward, received_amount, timestamp, vault = '0.000731782766782389', '3.396353893656228423', '0.001648465329300485', '248892.018071224099683254', TimestampMS(1742912819000), '0x464A190dc43aD8f706d7d90d2951F700226A47Ef'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            amount=FVal(gas_amount),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+            address=None,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=316,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=A_CRV,
+            amount=FVal(cvr_reward),
+            location_label=user_address,
+            notes=f'Claim {cvr_reward} CRV from StakeDAO vault',
+            counterparty=CPT_STAKEDAO,
+            address=vault,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=317,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=A_CVX,
+            amount=FVal(cvx_reward),
+            location_label=user_address,
+            notes=f'Claim {cvx_reward} CVX from StakeDAO vault',
+            counterparty=CPT_STAKEDAO,
+            address=vault,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=329,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REDEEM_WRAPPED,
+            asset=Asset('eip155:1/erc20:0x460638e6F7605B866736e38045C0DE8294d7D87f'),
+            amount=FVal(received_amount),
+            location_label=user_address,
+            notes=f'Withdraw {received_amount} alUSDsDOLA from StakeDAO vault',
+            counterparty=CPT_STAKEDAO,
+            address=vault,
+        ),
```

Comment:
aren't you missing the event that returns the gauge token back to the vault?

gauge token goes out ->
-> normal token comes in